rust/osm2lanes/src/tag/mod.rs

```diff
@@ -33,16 +34,12 @@ impl Tags {
         Ok(Self(map))
     }
 
-    // TODO deprecate, as it exposes the type of map used
-    pub fn new(map: BTreeMap<String, String>) -> Tags {
-        Tags(map)
-    }
-
-    // TODO deprecate, as it exposes the type of map used
-    // return something like &[[&str; 2]]
-    /// Expose inner map
-    pub fn map(&self) -> &BTreeMap<String, String> {
-        &self.0
+    /// Expose data as vector of pairs
+    pub fn to_str_pairs(&self) -> Vec<[&str; 2]> {
+        self.0
+            .iter()
+            .map(|(k, v)| [k.as_str(), v.as_str()])
+            .collect::<Vec<[&str; 2]>>()
```

Comment:
Type inference may let you do `collect::<Vec<_>>()` here and avoid repeating the return value

Reply:
```suggestion
            .collect::<Vec<_>>()
```